orionsdk/solarwinds.py

```diff
@@ -263,7 +263,48 @@ def add_icmp_echo_ip_sla_operation_to_node(self, node_name, ip_sla_operation_num
         results = self.swis.create('Orion.IpSla.Operations', **ip_sla_properties)
         self.logger.info("add_icmp_echo_ip_sla_operation_to_node - add IP SLA operation to node create results: %s", results)
 
-    def set_custom_properties(self, node_name, custom_property_name, custom_property_value):
```

Comment:
The method name changes for `set_custom_properties` and `get_custom_properties` are breaking changes, but those are better names.